api/api.go

```diff
@@ -83,3 +87,24 @@ func ReadJSONBody(body io.ReadCloser, v interface{}) error {
 
 	return nil
 }
+
+func (api *API) latestAPIRoutes(permissions AuthHandler) {
+	switch api.cfg.LatestVersion {
+	case "v1":
+		api.v1RoutesAsLatest(permissions)
+	default:
+		// Set default in case app configuration is set to non-existant version
+		api.v1RoutesAsLatest(permissions)
+	}
+}
+
+func (api *API) v1RoutesAsLatest(permissions AuthHandler) {
```

Comment:
It is ideal to not use version number in your function name as the version number for the latest will change in the future. I understand that you can create a new function in the future but this function can be more generic. This can be done by the previous comment